Xamarin.Forms.Platform.iOS/CollectionView/GridViewLayout.cs

```diff
@@ -144,6 +144,9 @@ bool NeedsPartialColumnAdjustment(int section = 0)
 				return false;
 			}
 
+			if (CollectionView.NumberOfSections() == 0)
```

Comment:
isnt't this from other PR ?

Reply:
The other PR took it from here. This wasn't ready to be merged yet, and I wanted to get that fix in because it was crashing.